src/modules/universal-discovery/universal.discovery.module.js

```diff
@@ -445,17 +512,16 @@ export default class UniversalDiscoveryModule extends Component {
         }
 
         const { contentTypesMap, maxHeight, activeTab, contentMeta } = this.state;
-        const isSelectedContentBookmarked = this.isBookmarked(contentMeta.id);
+        const isContentMetaBookmarked = this.isBookmarked(contentMeta.id);
```

Comment:
I don't like the new name. It states whether the content meta is bookmarked, not the content.